css-structure/css-structure_custom-properties/css/styles.css

```diff
@@ -8,6 +8,8 @@
   --color-black-light: #303030;
   --color-primary: #8236ff;
   --color-white: white;
```

Comment:
Consistent syntax for the color variables `--color-white=#ffffff`

Reply:
Oh, right, I forgot about that